src/bundle/Resources/translations/forms.en.xliff

```diff
@@ -556,201 +556,6 @@
         <target>Setup / System info</target>
         <note>key: role.policy.setup.system_info</note>
       </trans-unit>
-      <trans-unit id="029ea67634ff66aa9afd7c8850ebf6abe0f2e7be" resname="role.policy.siso_control_center">
-        <source>Siso control center</source>
-        <target>Siso control center</target>
-        <note>key: role.policy.siso_control_center</note>
-      </trans-unit>
-      <trans-unit id="ad4e634b7a562503b6e50d2b057490b8d7dc6336" resname="role.policy.siso_control_center.all_functions">
-        <source>Siso control center / All functions</source>
-        <target>Siso control center / All functions</target>
-        <note>key: role.policy.siso_control_center.all_functions</note>
```

Comment:
This probably should not be removed. It will remove translations in policy select.
cc @mikadamczyk